apps/accounts/templates/provider_portal/provider_domain_new/base.html

```diff
@@ -24,8 +24,17 @@ <h2 class="mt-4">
     <article>
       <div class="prose mx-auto mt-8">
         {% block explanatory_copy %}
-            <p class="italic">Explanatory copy here about how Carbon.txt gives you more control over how your provider is presented on Green Web Check results and Green Web  badges. </p>
-            <p>More copy explaining what is required (access to server, DNS, etc), and how long the process will take and what it will involve</p>
+            <p>Carbon.txt is a single recognisable location on any web domain for public sustainability data relating to that company. By combing Carbon.txt with your Green Web Foundation provider listing you can maintain a single source of truth for your organisations sustainability disclosures, and then share them across the different domains your organisation owns.</p>
+            <p>This benefit can also be passed on to your customers, who's domains can also be linked to your hosting provider listing our dataset. This ensures that whenever their domains are checked in our <a href="https://www.thegreenwebfoundation.org/green-web-check/" target="_blank">Green Web Check</a> and <a href="https://admin.thegreenwebfoundation.org/api-docs/" target="_blank">API</a> the results accurately reflect their use of your services.</p>
+
+            <p>To continue, you will need either:</p>
+
+            <ul>
+                <li>The ability to upload a Carbon.txt file to your website server. Or,</li>
+                <li>The ability to make DNS updates for the domain you wish to link to your provider. Or,</li>
+                <li>The ability to edit HTTP Headers for the domain you wish to link to your provider.</li>
+            </ul>
```

Comment:
I guess we're kinda glossing over the need to "have the information to hand to actually create the carbon.txt", but that's a bit of a can of worms that perhaps we should open at a later date